src/templates/utils-common.js

```diff
@@ -223,3 +223,22 @@ export function setLocaleCookie (locale, res, { useCookie, cookieDomain, cookieK
     res.setHeader('Set-Cookie', headers)
   }
 }
+
+/**
+ * @param  {string} pathString
+ * @param  {readonly string[]} localeCodes
+ * @return {string}
+ */
+export function clearPathFromLocale (pathString, localeCodes) {
```

Comment:
Naming feels off. Maybe
```suggestion
export function removeLocaleFromPath (pathString, localeCodes) {
```